python/its-interqueuemanager/its_iqm/authority/__init__.py

```diff
@@ -1,3 +1,25 @@
-import its_iqm.authority.file
-import its_iqm.authority.http
-import its_iqm.authority.mqtt
+from __future__ import annotations
+import its_iqm.authority.file as _file
+import its_iqm.authority.http as _http
+import its_iqm.authority.mqtt as _mqtt
+import its_iqm.iqm
+
+
+def Authority(
```

Comment:
As I explaioned in the commit log, it;s a function that looks liek a class, on purpose, so that isers can instantiate it.

In restrospect, that was probably a bad idea...

What about renaimg it to `get_authority_instance()` ?

Reply:
> > a function here: authority instead of Authority

> What about renaimg it to `get_authority_instance()` ?

After further thought thrown at it, I've made it a class, so kept it upper-case.